libraries/protocol/block.cpp

```diff
@@ -66,4 +66,12 @@ namespace steemit { namespace protocol {
       return checksum_type::hash( ids[0] );
    }
 
+   size_t signed_block::calculate_min_size()
```

Comment:
Because the method is static, the local variable, `b`, should also be static.

Reply:
Why?

Reply:
It is a static method and returns the same value every time. The returned value should also be static. Yes, we assert that a constant is equal to this value and then compare against the constant. Just because we are implementing this correctly now does not mean a future PR won't screw it up. These are things that easily slip through the cracks. I'd rather this method be done correctly and save us the hassle of potential problems in the future.